### ISSUE TITLE
[Legitimate Site Unblocked] etherspay.com

### ISSUE THREAD

**Issue by rayorole** [MetaMask/eth-phishing-detect#12540]
**Domain: etherspay.com**
[etherspay.com](https://etherspay.com)

**Details:** *Domain is a coming soon page for now, we are still in development. But we want to unblock the domain before we go live*

- [x] I have checked to make sure that [there is not a duplicate issue](https://github.com/MetaMask/eth-phishing-detect/issues)

**Comment by AlexHerman1** [comment]
we do not unblock domains where nothing is being hosted. please open a new issue when you have a project stood up here.
